### PR TITLE
Trello/4 copy button

### PR DESCRIPTION
**Pull request by ducdev**

### CODE REVIEW

File: imports/voting/ui/particles/SignStatement.vue [hunk 1]
```diff
@@ -124,18 +136,12 @@
...
-        left: 30px;
-      }
-      &.right {
-        bottom: 0;
-        right: 30px;
-      }
+    .btn-copy {
+      border: none;
+      margin: 0 auto;
+      display: flex;
```
Comment (davidyuk) [inline imports/voting/ui/particles/SignStatement.vue]: It should be just `block` instead of `flex`.

File: imports/voting/ui/particles/SignStatement.vue [hunk 2]
```diff
@@ -81,6 +87,12 @@
       }
     },
     methods: {
+      onCopyError() {
```
Comment (davidyuk) [inline imports/voting/ui/particles/SignStatement.vue]: Seem that `e` in arguments is missed.

File: imports/voting/ui/particles/SignStatement.vue [hunk 3]
```diff
@@ -53,6 +50,14 @@
...
       </form>
     </div>
+    <h5>Share this link</h5>
+    <span>{{`http://voting.aepps.com/proposal/${proposalId}`}}</span>
+    <button
+      class="btn-copy"
+      v-clipboard="`http://voting.aepps.com/proposal/${proposalId}`"
+      @success="onCopied"
+      @error="onCopyError"
+    >Copy</button>
```
Comment (davidyuk) [inline imports/voting/ui/particles/SignStatement.vue:62]: It should be a part of `ProposalDetail` page because `SignStatement` is also used in `CreateProposalModal`.

File: imports/voting/startup/client/index.js [hunk 4]
```diff
@@ -1,12 +1,12 @@
 import Vue from 'vue';
 import VueRouter from 'vue-router';
 import Vuex from 'vuex';
+import VueClipboards from 'vue-clipboards';
 import { sync } from 'vuex-router-sync';
 import VueMeteorTracker from 'vue-meteor-tracker';
 import { injectSupply } from 'vue-supply';
 import VueAutosize from 'vue-autosize';
 import createPersistedState from 'vuex-persistedstate';
-
```
Comment (davidyuk) [inline imports/voting/startup/client/index.js]: Why have you removed this line? Try don't commit changes that is not related to topic of current commit (77835d8856498aa4ae076fcb716156bc3fd103ac).

File: imports/voting/ui/particles/SignStatement.vue [hunk 5]
```diff
@@ -157,7 +163,7 @@
   .voting-section {
     &.doubt {
       .tab {
-        background: $red;
+        background: $red !important;
```
Comment (davidyuk) [inline imports/voting/ui/particles/SignStatement.vue:169]: I believe that there is another way without using of `!Important`.

**Review by davidyuk — CHANGES_REQUESTED** [review]
If you'll split this set of changes in separate PR's, we'll be able to merge them separately.

File: imports/voting/ui/layouts/MainLayout.vue [hunk 6]
```diff
@@ -45,9 +44,10 @@
   .main-content {
     max-width: $container-width;
     margin: 0 auto;
-    padding-top: 10px;
+    padding-top: 115px;
```
[the remark this answers is not in the document]
Comment (ducdev) [reply imports/voting/ui/layouts/MainLayout.vue:47]: For me it's working, you please help me to make your approach works with many browsers or we can use padding approach for easier ?

File: imports/voting/ui/particles/SignStatement.vue (hunk 5 above)
Comment (ducdev) [reply imports/voting/ui/particles/SignStatement.vue:169]: Which way man ?

File: imports/voting/ui/particles/SignStatement.vue (hunk 3 above)
Comment (ducdev) [reply imports/voting/ui/particles/SignStatement.vue:62]: How possible, because it's below the `{{ statement }}` which is inside `SignStatement.vue`

Comment (ducdev) [reply imports/voting/ui/particles/SignStatement.vue:62]: I guess we need a prop to tell the component when it should show the copy button or not.
